libfuse/folderlist.go

```diff
@@ -146,6 +151,11 @@ func (fl *FolderList) Mkdir(ctx context.Context, req *fuse.MkdirRequest) (_ fs.N
 	fl.fs.log.CDebugf(ctx, "FL Mkdir")
 	tlfName := libkbfs.CanonicalTlfName(req.Name)
 	defer func() { fl.reportErr(ctx, libkbfs.WriteMode, tlfName, err) }()
+	if strings.HasPrefix(req.Name, "._") {
```

Comment:
Yes, since that would be an intentional operation, rather than macOS kernel. The latter could happen when user does stuff in Finder on legitimate directories, but the former can only be explicitly done by user (or a program)